lib/spacesuit.ex

```diff
@@ -15,7 +15,7 @@ defmodule Spacesuit do
     dispatch = Spacesuit.Router.load_routes() |> :cowboy_router.compile()
 
     {:ok, _} =
-      :cowboy.start_clear(:http, 600, [port: @http_port], %{
+      :cowboy.start_clear(:http, [port: @http_port, num_acceptors: 600], %{
```

Comment:
so while this allows the code to build, I'm pretty certain this isn't correct as I've indicated in my original comments on the PR. you'll need to sort this out before merging. I.e., `num_acceptors` doesn't appear to be a valid option any longer, and changing the port doesn't affect the server, meaning it always starts on `8080`, so I think this needs further attention.